src/main/java/com/twilio/http/bearertoken/BearerTokenTwilioRestClient.java

```diff
@@ -103,9 +112,9 @@ public Response request(BearerTokenRequest request) {
         if (accessToken == null || accessToken.isEmpty()) {
             throw new AuthenticationException("Access Token can not be Null or Empty.");
```

Comment:
This code will throw **AuthenticationException** every time, since accessToken in empty at first go.